webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala

```diff
@@ -374,24 +399,22 @@ class TestNodeAndGlobalParameterLookup extends Specification {
     }
 
     "parse text and variable and text" in {
-      val s1 = "plj jmoji h imj "
-      val s2 = " alkjf fm ^{i àié" // will be split at '$'
-      val s3 = "${rudde ut ùt "
+      val s1 = "plj jmoji h imj " // 16
+      val s2 = " alkjf fm ^{i àié$rudde ut ùt ending"
       test(
         all(_),
-        s1 + "${rudder.node.policyserver.id}" + s2 + s3,
-        List(CharSeq(s1), NodeAccessor(List("policyserver", "id")), CharSeq(s2), CharSeq(s3))
+        s1 + "${rudder.node.policyserver.id}" + s2,
+        List(CharSeq(s1), NodeAccessor(List("policyserver", "id")), CharSeq(s2))
```

Comment:
SPEC has changed : since we changed the parser and introduced a `UnsafeRudderVar` parser, we consider that `${` should always be followed by the enclosing `}`, or else we will end up with a parsing error